[29/05/2023 05:01:52 p. m.] Error: System.IndexOutOfRangeException: Could not find specified column in results: Tipo_pago
   at MySql.Data.MySqlClient.ResultSet.GetOrdinal(String name)
   at MySql.Data.MySqlClient.MySqlDataReader.GetString(String column)
   at FutureLending.Lectura_Base_Datos.LectTodos(String tabla, String lista) in D:\Documentos\Trabajos\FutureLendingpro\Lectura_Base_Datos.cs:line 312
[29/05/2023 05:01:52 p. m.] Error: System.IndexOutOfRangeException: Could not find specified column in results: Tipo_pago
   at MySql.Data.MySqlClient.ResultSet.GetOrdinal(String name)
   at MySql.Data.MySqlClient.MySqlDataReader.GetString(String column)
   at FutureLending.Lectura_Base_Datos.LectTodos(String tabla, String lista) in D:\Documentos\Trabajos\FutureLendingpro\Lectura_Base_Datos.cs:line 312
[29/05/2023 05:04:44 p. m.] Error: System.IndexOutOfRangeException: Could not find specified column in results: Tipo_pago
   at MySql.Data.MySqlClient.ResultSet.GetOrdinal(String name)
   at MySql.Data.MySqlClient.MySqlDataReader.GetString(String column)
   at FutureLending.Lectura_Base_Datos.LectTodos(String tabla, String lista) in D:\Documentos\Trabajos\FutureLendingpro\Lectura_Base_Datos.cs:line 312
[30/05/2023 05:45:25 a. m.] Error: MySql.Data.MySqlClient.MySqlException (0x80004005): Unable to connect to any of the specified MySQL hosts.
 ---> System.Net.Sockets.SocketException (10061): No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión.
   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.ThrowException(SocketError error, CancellationToken cancellationToken)
   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.System.Threading.Tasks.Sources.IValueTaskSource.GetResult(Int16 token)
   at System.Threading.Tasks.ValueTask.ValueTaskSourceAsTask.<>c.<.cctor>b__4_0(Object state)
--- End of stack trace from previous location ---
   at System.Net.Sockets.TcpClient.CompleteConnectAsync(Task task)
   at MySql.Data.MySqlClient.NativeDriver.<>c.<OpenAsync>b__38_0(Exception ex)
   at System.AggregateException.Handle(Func`2 predicate)
   at MySql.Data.MySqlClient.NativeDriver.OpenAsync(Boolean execAsync, CancellationToken cancellationToken)
   at MySql.Data.MySqlClient.Driver.OpenAsync(Boolean execAsync, CancellationToken cancellationToken)
   at MySql.Data.MySqlClient.Driver.CreateAsync(MySqlConnectionStringBuilder settings, Boolean execAsync, CancellationToken cancellationToken)
   at MySql.Data.MySqlClient.Driver.CreateAsync(MySqlConnectionStringBuilder settings, Boolean execAsync, CancellationToken cancellationToken)
   at MySql.Data.MySqlClient.MySqlPool.CreateNewPooledConnectionAsync(Boolean execAsync, CancellationToken cancellationToken)
   at MySql.Data.MySqlClient.MySqlPool.GetPooledConnectionAsync(Boolean execAsync, CancellationToken cancellationToken)
   at MySql.Data.MySqlClient.MySqlPool.TryToGetDriverAsync(Boolean execAsync, CancellationToken cancellationToken)
   at MySql.Data.MySqlClient.MySqlPool.GetConnectionAsync(Boolean execAsync, CancellationToken cancellationToken)
   at MySql.Data.MySqlClient.MySqlConnection.OpenAsync(Boolean execAsync, CancellationToken cancellationToken)
   at MySql.Data.MySqlClient.MySqlConnection.Open()
   at FutureLending.Lectura_Base_Datos.Conector() in D:\Documentos\Trabajos\FutureLendingpro\Lectura_Base_Datos.cs:line 57
[30/05/2023 05:45:25 a. m.] Error: System.InvalidOperationException: Connection must be valid and open.
   at MySql.Data.MySqlClient.Interceptors.ExceptionInterceptor.Throw(Exception exception)
   at MySql.Data.MySqlClient.MySqlCommand.CheckState()
   at MySql.Data.MySqlClient.MySqlCommand.ExecuteReaderAsync(CommandBehavior behavior, Boolean execAsync, CancellationToken cancellationToken)
   at MySql.Data.MySqlClient.MySqlCommand.ExecuteReader()
   at FutureLending.Lectura_Base_Datos.LectLista1() in D:\Documentos\Trabajos\FutureLendingpro\Lectura_Base_Datos.cs:line 95
[30/05/2023 05:45:28 a. m.] Error: MySql.Data.MySqlClient.MySqlException (0x80004005): Unable to connect to any of the specified MySQL hosts.
 ---> System.Net.Sockets.SocketException (10061): No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión.
   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.ThrowException(SocketError error, CancellationToken cancellationToken)
   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.System.Threading.Tasks.Sources.IValueTaskSource.GetResult(Int16 token)
   at System.Threading.Tasks.ValueTask.ValueTaskSourceAsTask.<>c.<.cctor>b__4_0(Object state)
--- End of stack trace from previous location ---
   at System.Net.Sockets.TcpClient.CompleteConnectAsync(Task task)
   at MySql.Data.MySqlClient.NativeDriver.<>c.<OpenAsync>b__38_0(Exception ex)
   at System.AggregateException.Handle(Func`2 predicate)
   at MySql.Data.MySqlClient.NativeDriver.OpenAsync(Boolean execAsync, CancellationToken cancellationToken)
   at MySql.Data.MySqlClient.Driver.OpenAsync(Boolean execAsync, CancellationToken cancellationToken)
   at MySql.Data.MySqlClient.Driver.CreateAsync(MySqlConnectionStringBuilder settings, Boolean execAsync, CancellationToken cancellationToken)
   at MySql.Data.MySqlClient.Driver.CreateAsync(MySqlConnectionStringBuilder settings, Boolean execAsync, CancellationToken cancellationToken)
   at MySql.Data.MySqlClient.MySqlPool.CreateNewPooledConnectionAsync(Boolean execAsync, CancellationToken cancellationToken)
   at MySql.Data.MySqlClient.MySqlPool.GetPooledConnectionAsync(Boolean execAsync, CancellationToken cancellationToken)
   at MySql.Data.MySqlClient.MySqlPool.TryToGetDriverAsync(Boolean execAsync, CancellationToken cancellationToken)
   at MySql.Data.MySqlClient.MySqlPool.GetConnectionAsync(Boolean execAsync, CancellationToken cancellationToken)
   at MySql.Data.MySqlClient.MySqlConnection.OpenAsync(Boolean execAsync, CancellationToken cancellationToken)
   at MySql.Data.MySqlClient.MySqlConnection.Open()
   at FutureLending.Lectura_Base_Datos.Conector() in D:\Documentos\Trabajos\FutureLendingpro\Lectura_Base_Datos.cs:line 57
[30/05/2023 05:45:28 a. m.] Error: System.InvalidOperationException: Connection must be valid and open.
   at MySql.Data.MySqlClient.Interceptors.ExceptionInterceptor.Throw(Exception exception)
   at MySql.Data.MySqlClient.MySqlCommand.CheckState()
   at MySql.Data.MySqlClient.MySqlCommand.ExecuteReaderAsync(CommandBehavior behavior, Boolean execAsync, CancellationToken cancellationToken)
   at MySql.Data.MySqlClient.MySqlCommand.ExecuteReader()
   at FutureLending.Lectura_Base_Datos.LectLista2() in D:\Documentos\Trabajos\FutureLendingpro\Lectura_Base_Datos.cs:line 147
[30/05/2023 05:45:32 a. m.] Error: MySql.Data.MySqlClient.MySqlException (0x80004005): Unable to connect to any of the specified MySQL hosts.
 ---> System.Net.Sockets.SocketException (10061): No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión.
   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.ThrowException(SocketError error, CancellationToken cancellationToken)
   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.System.Threading.Tasks.Sources.IValueTaskSource.GetResult(Int16 token)
   at System.Threading.Tasks.ValueTask.ValueTaskSourceAsTask.<>c.<.cctor>b__4_0(Object state)
--- End of stack trace from previous location ---
   at System.Net.Sockets.TcpClient.CompleteConnectAsync(Task task)
   at MySql.Data.Common.StreamCreator.GetTcpStreamAsync(MySqlConnectionStringBuilder settings, CancellationToken cancellationToken, Boolean execAsync)
   at MySql.Data.Common.StreamCreator.GetStreamAsync(MySqlConnectionStringBuilder settings, CancellationToken cancellationToken, Boolean execAsync)
   at MySql.Data.MySqlClient.NativeDriver.OpenAsync(Boolean execAsync, CancellationToken cancellationToken)
   at MySql.Data.MySqlClient.NativeDriver.OpenAsync(Boolean execAsync, CancellationToken cancellationToken)
   at MySql.Data.MySqlClient.Driver.OpenAsync(Boolean execAsync, CancellationToken cancellationToken)
   at MySql.Data.MySqlClient.Driver.CreateAsync(MySqlConnectionStringBuilder settings, Boolean execAsync, CancellationToken cancellationToken)
   at MySql.Data.MySqlClient.Driver.CreateAsync(MySqlConnectionStringBuilder settings, Boolean execAsync, CancellationToken cancellationToken)
   at MySql.Data.MySqlClient.MySqlPool.CreateNewPooledConnectionAsync(Boolean execAsync, CancellationToken cancellationToken)
   at MySql.Data.MySqlClient.MySqlPool.GetPooledConnectionAsync(Boolean execAsync, CancellationToken cancellationToken)
   at MySql.Data.MySqlClient.MySqlPool.TryToGetDriverAsync(Boolean execAsync, CancellationToken cancellationToken)
   at MySql.Data.MySqlClient.MySqlPool.GetConnectionAsync(Boolean execAsync, CancellationToken cancellationToken)
   at MySql.Data.MySqlClient.MySqlConnection.OpenAsync(Boolean execAsync, CancellationToken cancellationToken)
   at FutureLending.Lectura_Base_Datos.CheckConnection(Boolean revisador) in D:\Documentos\Trabajos\FutureLendingpro\Lectura_Base_Datos.cs:line 629
[30/05/2023 05:45:41 a. m.] Error: System.IndexOutOfRangeException: Could not find specified column in results: Tipo_pago
   at MySql.Data.MySqlClient.ResultSet.GetOrdinal(String name)
   at MySql.Data.MySqlClient.MySqlDataReader.GetString(String column)
   at FutureLending.Lectura_Base_Datos.LectTodos(String tabla, String lista) in D:\Documentos\Trabajos\FutureLendingpro\Lectura_Base_Datos.cs:line 312
[30/05/2023 05:45:43 a. m.] Error: System.IndexOutOfRangeException: Could not find specified column in results: Tipo_pago
   at MySql.Data.MySqlClient.ResultSet.GetOrdinal(String name)
   at MySql.Data.MySqlClient.MySqlDataReader.GetString(String column)
   at FutureLending.Lectura_Base_Datos.LectTodos(String tabla, String lista) in D:\Documentos\Trabajos\FutureLendingpro\Lectura_Base_Datos.cs:line 312
[30/05/2023 06:16:34 a. m.] Error: MySql.Data.MySqlClient.MySqlException (0x80004005): Unable to connect to any of the specified MySQL hosts.
 ---> System.Net.Sockets.SocketException (10061): No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión.
   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.ThrowException(SocketError error, CancellationToken cancellationToken)
   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.System.Threading.Tasks.Sources.IValueTaskSource.GetResult(Int16 token)
   at System.Threading.Tasks.ValueTask.ValueTaskSourceAsTask.<>c.<.cctor>b__4_0(Object state)
--- End of stack trace from previous location ---
   at System.Net.Sockets.TcpClient.CompleteConnectAsync(Task task)
   at MySql.Data.MySqlClient.NativeDriver.<>c.<OpenAsync>b__38_0(Exception ex)
   at System.AggregateException.Handle(Func`2 predicate)
   at MySql.Data.MySqlClient.NativeDriver.OpenAsync(Boolean execAsync, CancellationToken cancellationToken)
   at MySql.Data.MySqlClient.Driver.OpenAsync(Boolean execAsync, CancellationToken cancellationToken)
   at MySql.Data.MySqlClient.Driver.CreateAsync(MySqlConnectionStringBuilder settings, Boolean execAsync, CancellationToken cancellationToken)
   at MySql.Data.MySqlClient.Driver.CreateAsync(MySqlConnectionStringBuilder settings, Boolean execAsync, CancellationToken cancellationToken)
   at MySql.Data.MySqlClient.MySqlPool.CreateNewPooledConnectionAsync(Boolean execAsync, CancellationToken cancellationToken)
   at MySql.Data.MySqlClient.MySqlPool.GetPooledConnectionAsync(Boolean execAsync, CancellationToken cancellationToken)
   at MySql.Data.MySqlClient.MySqlPool.TryToGetDriverAsync(Boolean execAsync, CancellationToken cancellationToken)
   at MySql.Data.MySqlClient.MySqlPool.GetConnectionAsync(Boolean execAsync, CancellationToken cancellationToken)
   at MySql.Data.MySqlClient.MySqlConnection.OpenAsync(Boolean execAsync, CancellationToken cancellationToken)
   at MySql.Data.MySqlClient.MySqlConnection.Open()
   at FutureLending.Lectura_Base_Datos.Conector() in D:\Documentos\Trabajos\FutureLendingpro\Lectura_Base_Datos.cs:line 57
[30/05/2023 06:16:34 a. m.] Error: System.InvalidOperationException: Connection must be valid and open.
   at MySql.Data.MySqlClient.Interceptors.ExceptionInterceptor.Throw(Exception exception)
   at MySql.Data.MySqlClient.MySqlCommand.CheckState()
   at MySql.Data.MySqlClient.MySqlCommand.ExecuteReaderAsync(CommandBehavior behavior, Boolean execAsync, CancellationToken cancellationToken)
   at MySql.Data.MySqlClient.MySqlCommand.ExecuteReader()
   at FutureLending.Lectura_Base_Datos.LectLista2() in D:\Documentos\Trabajos\FutureLendingpro\Lectura_Base_Datos.cs:line 147
[30/05/2023 06:16:37 a. m.] Error: MySql.Data.MySqlClient.MySqlException (0x80004005): Unable to connect to any of the specified MySQL hosts.
 ---> System.Net.Sockets.SocketException (10061): No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión.
   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.ThrowException(SocketError error, CancellationToken cancellationToken)
   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.System.Threading.Tasks.Sources.IValueTaskSource.GetResult(Int16 token)
   at System.Threading.Tasks.ValueTask.ValueTaskSourceAsTask.<>c.<.cctor>b__4_0(Object state)
--- End of stack trace from previous location ---
   at System.Net.Sockets.TcpClient.CompleteConnectAsync(Task task)
   at MySql.Data.Common.StreamCreator.GetTcpStreamAsync(MySqlConnectionStringBuilder settings, CancellationToken cancellationToken, Boolean execAsync)
   at MySql.Data.Common.StreamCreator.GetStreamAsync(MySqlConnectionStringBuilder settings, CancellationToken cancellationToken, Boolean execAsync)
   at MySql.Data.MySqlClient.NativeDriver.OpenAsync(Boolean execAsync, CancellationToken cancellationToken)
   at MySql.Data.MySqlClient.NativeDriver.OpenAsync(Boolean execAsync, CancellationToken cancellationToken)
   at MySql.Data.MySqlClient.Driver.OpenAsync(Boolean execAsync, CancellationToken cancellationToken)
   at MySql.Data.MySqlClient.Driver.CreateAsync(MySqlConnectionStringBuilder settings, Boolean execAsync, CancellationToken cancellationToken)
   at MySql.Data.MySqlClient.Driver.CreateAsync(MySqlConnectionStringBuilder settings, Boolean execAsync, CancellationToken cancellationToken)
   at MySql.Data.MySqlClient.MySqlPool.CreateNewPooledConnectionAsync(Boolean execAsync, CancellationToken cancellationToken)
   at MySql.Data.MySqlClient.MySqlPool.GetPooledConnectionAsync(Boolean execAsync, CancellationToken cancellationToken)
   at MySql.Data.MySqlClient.MySqlPool.TryToGetDriverAsync(Boolean execAsync, CancellationToken cancellationToken)
   at MySql.Data.MySqlClient.MySqlPool.GetConnectionAsync(Boolean execAsync, CancellationToken cancellationToken)
   at MySql.Data.MySqlClient.MySqlConnection.OpenAsync(Boolean execAsync, CancellationToken cancellationToken)
   at FutureLending.Lectura_Base_Datos.CheckConnection(Boolean revisador) in D:\Documentos\Trabajos\FutureLendingpro\Lectura_Base_Datos.cs:line 629
[30/05/2023 07:56:24 p. m.] Error: MySql.Data.MySqlClient.MySqlException (0x80004005): Unable to connect to any of the specified MySQL hosts.
 ---> System.Net.Sockets.SocketException (10061): No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión.
   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.ThrowException(SocketError error, CancellationToken cancellationToken)
   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.System.Threading.Tasks.Sources.IValueTaskSource.GetResult(Int16 token)
   at System.Threading.Tasks.ValueTask.ValueTaskSourceAsTask.<>c.<.cctor>b__4_0(Object state)
--- End of stack trace from previous location ---
   at System.Net.Sockets.TcpClient.CompleteConnectAsync(Task task)
   at MySql.Data.MySqlClient.NativeDriver.<>c.<OpenAsync>b__38_0(Exception ex)
   at System.AggregateException.Handle(Func`2 predicate)
   at MySql.Data.MySqlClient.NativeDriver.OpenAsync(Boolean execAsync, CancellationToken cancellationToken)
   at MySql.Data.MySqlClient.Driver.OpenAsync(Boolean execAsync, CancellationToken cancellationToken)
   at MySql.Data.MySqlClient.Driver.CreateAsync(MySqlConnectionStringBuilder settings, Boolean execAsync, CancellationToken cancellationToken)
   at MySql.Data.MySqlClient.Driver.CreateAsync(MySqlConnectionStringBuilder settings, Boolean execAsync, CancellationToken cancellationToken)
   at MySql.Data.MySqlClient.MySqlPool.CreateNewPooledConnectionAsync(Boolean execAsync, CancellationToken cancellationToken)
   at MySql.Data.MySqlClient.MySqlPool.GetPooledConnectionAsync(Boolean execAsync, CancellationToken cancellationToken)
   at MySql.Data.MySqlClient.MySqlPool.TryToGetDriverAsync(Boolean execAsync, CancellationToken cancellationToken)
   at MySql.Data.MySqlClient.MySqlPool.GetConnectionAsync(Boolean execAsync, CancellationToken cancellationToken)
   at MySql.Data.MySqlClient.MySqlConnection.OpenAsync(Boolean execAsync, CancellationToken cancellationToken)
   at MySql.Data.MySqlClient.MySqlConnection.Open()
   at FutureLending.Lectura_Base_Datos.Conector() in D:\Documentos\Trabajos\FutureLendingpro\Lectura_Base_Datos.cs:line 57
[30/05/2023 07:56:24 p. m.] Error: System.InvalidOperationException: Connection must be valid and open.
   at MySql.Data.MySqlClient.Interceptors.ExceptionInterceptor.Throw(Exception exception)
   at MySql.Data.MySqlClient.MySqlCommand.CheckState()
   at MySql.Data.MySqlClient.MySqlCommand.ExecuteReaderAsync(CommandBehavior behavior, Boolean execAsync, CancellationToken cancellationToken)
   at MySql.Data.MySqlClient.MySqlCommand.ExecuteScalarAsync(Boolean execAsync, CancellationToken cancellationToken)
   at MySql.Data.MySqlClient.MySqlCommand.ExecuteScalar()
   at FutureLending.Accesos.Accesar(String user, String password) in D:\Documentos\Trabajos\FutureLendingpro\Accesos.cs:line 25
[30/05/2023 07:56:29 p. m.] Error: MySql.Data.MySqlClient.MySqlException (0x80004005): Unable to connect to any of the specified MySQL hosts.
 ---> System.Net.Sockets.SocketException (10061): No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión.
   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.ThrowException(SocketError error, CancellationToken cancellationToken)
   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.System.Threading.Tasks.Sources.IValueTaskSource.GetResult(Int16 token)
   at System.Threading.Tasks.ValueTask.ValueTaskSourceAsTask.<>c.<.cctor>b__4_0(Object state)
--- End of stack trace from previous location ---
   at System.Net.Sockets.TcpClient.CompleteConnectAsync(Task task)
   at MySql.Data.Common.StreamCreator.GetTcpStreamAsync(MySqlConnectionStringBuilder settings, CancellationToken cancellationToken, Boolean execAsync)
   at MySql.Data.Common.StreamCreator.GetStreamAsync(MySqlConnectionStringBuilder settings, CancellationToken cancellationToken, Boolean execAsync)
   at MySql.Data.MySqlClient.NativeDriver.OpenAsync(Boolean execAsync, CancellationToken cancellationToken)
   at MySql.Data.MySqlClient.NativeDriver.OpenAsync(Boolean execAsync, CancellationToken cancellationToken)
   at MySql.Data.MySqlClient.Driver.OpenAsync(Boolean execAsync, CancellationToken cancellationToken)
   at MySql.Data.MySqlClient.Driver.CreateAsync(MySqlConnectionStringBuilder settings, Boolean execAsync, CancellationToken cancellationToken)
   at MySql.Data.MySqlClient.Driver.CreateAsync(MySqlConnectionStringBuilder settings, Boolean execAsync, CancellationToken cancellationToken)
   at MySql.Data.MySqlClient.MySqlPool.CreateNewPooledConnectionAsync(Boolean execAsync, CancellationToken cancellationToken)
   at MySql.Data.MySqlClient.MySqlPool.GetPooledConnectionAsync(Boolean execAsync, CancellationToken cancellationToken)
   at MySql.Data.MySqlClient.MySqlPool.TryToGetDriverAsync(Boolean execAsync, CancellationToken cancellationToken)
   at MySql.Data.MySqlClient.MySqlPool.GetConnectionAsync(Boolean execAsync, CancellationToken cancellationToken)
   at MySql.Data.MySqlClient.MySqlConnection.OpenAsync(Boolean execAsync, CancellationToken cancellationToken)
   at FutureLending.Lectura_Base_Datos.CheckConnection(Boolean revisador) in D:\Documentos\Trabajos\FutureLendingpro\Lectura_Base_Datos.cs:line 629
[30/05/2023 07:56:46 p. m.] Error: System.IndexOutOfRangeException: Could not find specified column in results: Tipo_pago
   at MySql.Data.MySqlClient.ResultSet.GetOrdinal(String name)
   at MySql.Data.MySqlClient.MySqlDataReader.GetString(String column)
   at FutureLending.Lectura_Base_Datos.LectTodos(String tabla, String lista) in D:\Documentos\Trabajos\FutureLendingpro\Lectura_Base_Datos.cs:line 312
[31/05/2023 11:33:37 a. m.] Error: System.IndexOutOfRangeException: Could not find specified column in results: Tipo_pago
   at MySql.Data.MySqlClient.ResultSet.GetOrdinal(String name)
   at MySql.Data.MySqlClient.MySqlDataReader.GetString(String column)
   at FutureLending.Lectura_Base_Datos.LectTodos(String tabla, String lista) in D:\FutureLendingpro\Lectura_Base_Datos.cs:line 312
[31/05/2023 11:34:39 a. m.] Error: System.IndexOutOfRangeException: Could not find specified column in results: Tipo_pago
   at MySql.Data.MySqlClient.ResultSet.GetOrdinal(String name)
   at MySql.Data.MySqlClient.MySqlDataReader.GetString(String column)
   at FutureLending.Lectura_Base_Datos.LectTodos(String tabla, String lista) in D:\FutureLendingpro\Lectura_Base_Datos.cs:line 312
[31/05/2023 08:45:44 p. m.] Error: Error al cargar los promotores: Operación no válida a través de subprocesos: Se tuvo acceso al control '' desde un subproceso distinto a aquel en que lo creó.
[31/05/2023 08:46:45 p. m.] Error: Error al cargar los promotores: Operación no válida a través de subprocesos: Se tuvo acceso al control '' desde un subproceso distinto a aquel en que lo creó.
[31/05/2023 08:46:45 p. m.] Error: Error al cargar los promotores: Operación no válida a través de subprocesos: Se tuvo acceso al control '' desde un subproceso distinto a aquel en que lo creó.
[31/05/2023 08:58:28 p. m.] Error: Error al cargar los promotores: Operación no válida a través de subprocesos: Se tuvo acceso al control '' desde un subproceso distinto a aquel en que lo creó.
[31/05/2023 08:58:28 p. m.] Error: Error al cargar los promotores: Operación no válida a través de subprocesos: Se tuvo acceso al control '' desde un subproceso distinto a aquel en que lo creó.
[31/05/2023 09:00:33 p. m.] Error: Error al cargar los promotores: Operación no válida a través de subprocesos: Se tuvo acceso al control '' desde un subproceso distinto a aquel en que lo creó.
[31/05/2023 09:00:33 p. m.] Error: Error al cargar los promotores: Operación no válida a través de subprocesos: Se tuvo acceso al control '' desde un subproceso distinto a aquel en que lo creó.
[31/05/2023 09:00:58 p. m.] Error: System.IndexOutOfRangeException: Could not find specified column in results: Tipo_pago
   at MySql.Data.MySqlClient.ResultSet.GetOrdinal(String name)
   at MySql.Data.MySqlClient.MySqlDataReader.GetString(String column)
   at FutureLending.Lectura_Base_Datos.LectTodos(String tabla, String lista) in D:\Escritorio\FutureLendingpro\Funciones.cs\Lectura_Base_Datos.cs:line 312
[31/05/2023 09:04:40 p. m.] Error: Error al cargar los promotores: Operación no válida a través de subprocesos: Se tuvo acceso al control '' desde un subproceso distinto a aquel en que lo creó.
[31/05/2023 09:04:40 p. m.] Error: Error al cargar los promotores: Operación no válida a través de subprocesos: Se tuvo acceso al control '' desde un subproceso distinto a aquel en que lo creó.
[31/05/2023 09:05:53 p. m.] Error: Error al cargar los promotores: Operación no válida a través de subprocesos: Se tuvo acceso al control '' desde un subproceso distinto a aquel en que lo creó.
[31/05/2023 09:05:53 p. m.] Error: Error al cargar los promotores: Operación no válida a través de subprocesos: Se tuvo acceso al control '' desde un subproceso distinto a aquel en que lo creó.
[31/05/2023 09:08:37 p. m.] Error: Error al cargar los promotores: Operación no válida a través de subprocesos: Se tuvo acceso al control '' desde un subproceso distinto a aquel en que lo creó.
[31/05/2023 09:08:37 p. m.] Error: Error al cargar los promotores: Operación no válida a través de subprocesos: Se tuvo acceso al control '' desde un subproceso distinto a aquel en que lo creó.
[31/05/2023 09:09:31 p. m.] Error: Error al cargar los promotores: Operación no válida a través de subprocesos: Se tuvo acceso al control '' desde un subproceso distinto a aquel en que lo creó.
[31/05/2023 09:09:31 p. m.] Error: Error al cargar los promotores: Operación no válida a través de subprocesos: Se tuvo acceso al control '' desde un subproceso distinto a aquel en que lo creó.
[31/05/2023 09:10:34 p. m.] Error: Error al cargar los promotores: Operación no válida a través de subprocesos: Se tuvo acceso al control '' desde un subproceso distinto a aquel en que lo creó.
[31/05/2023 09:10:34 p. m.] Error: Error al cargar los promotores: Operación no válida a través de subprocesos: Se tuvo acceso al control '' desde un subproceso distinto a aquel en que lo creó.
[31/05/2023 09:10:38 p. m.] Error: System.IndexOutOfRangeException: Could not find specified column in results: Tipo_pago
   at MySql.Data.MySqlClient.ResultSet.GetOrdinal(String name)
   at MySql.Data.MySqlClient.MySqlDataReader.GetString(String column)
   at FutureLending.Lectura_Base_Datos.LectTodos(String tabla, String lista) in D:\Escritorio\FutureLendingpro\Funciones.cs\Lectura_Base_Datos.cs:line 312
[31/05/2023 09:13:54 p. m.] Error: Error al cargar los promotores: Operación no válida a través de subprocesos: Se tuvo acceso al control '' desde un subproceso distinto a aquel en que lo creó.
[31/05/2023 09:13:54 p. m.] Error: Error al cargar los promotores: Operación no válida a través de subprocesos: Se tuvo acceso al control '' desde un subproceso distinto a aquel en que lo creó.
[31/05/2023 09:15:21 p. m.] Error: Error al cargar los promotores: Operación no válida a través de subprocesos: Se tuvo acceso al control '' desde un subproceso distinto a aquel en que lo creó.
[31/05/2023 09:15:21 p. m.] Error: Error al cargar los promotores: Operación no válida a través de subprocesos: Se tuvo acceso al control '' desde un subproceso distinto a aquel en que lo creó.
[31/05/2023 09:16:08 p. m.] Error: Error al cargar los promotores: Operación no válida a través de subprocesos: Se tuvo acceso al control '' desde un subproceso distinto a aquel en que lo creó.
[31/05/2023 09:16:08 p. m.] Error: Error al cargar los promotores: Operación no válida a través de subprocesos: Se tuvo acceso al control '' desde un subproceso distinto a aquel en que lo creó.
[31/05/2023 09:18:26 p. m.] Error: Error al cargar los promotores: Operación no válida a través de subprocesos: Se tuvo acceso al control '' desde un subproceso distinto a aquel en que lo creó.
[31/05/2023 09:18:26 p. m.] Error: Error al cargar los promotores: Operación no válida a través de subprocesos: Se tuvo acceso al control '' desde un subproceso distinto a aquel en que lo creó.
[31/05/2023 09:20:00 p. m.] Error: Error al cargar los promotores: Operación no válida a través de subprocesos: Se tuvo acceso al control '' desde un subproceso distinto a aquel en que lo creó.
[31/05/2023 09:20:00 p. m.] Error: Error al cargar los promotores: Operación no válida a través de subprocesos: Se tuvo acceso al control '' desde un subproceso distinto a aquel en que lo creó.
[31/05/2023 09:25:25 p. m.] Error: Error al cargar los promotores: Operación no válida a través de subprocesos: Se tuvo acceso al control '' desde un subproceso distinto a aquel en que lo creó.
[31/05/2023 09:25:25 p. m.] Error: Error al cargar los promotores: Operación no válida a través de subprocesos: Se tuvo acceso al control '' desde un subproceso distinto a aquel en que lo creó.
[31/05/2023 09:27:34 p. m.] Error: Error al cargar los promotores: Operación no válida a través de subprocesos: Se tuvo acceso al control '' desde un subproceso distinto a aquel en que lo creó.
[31/05/2023 09:27:34 p. m.] Error: Error al cargar los promotores: Operación no válida a través de subprocesos: Se tuvo acceso al control '' desde un subproceso distinto a aquel en que lo creó.
[31/05/2023 09:28:26 p. m.] Error: System.IndexOutOfRangeException: Could not find specified column in results: Tipo_pago
   at MySql.Data.MySqlClient.ResultSet.GetOrdinal(String name)
   at MySql.Data.MySqlClient.MySqlDataReader.GetString(String column)
   at FutureLending.Lectura_Base_Datos.LectTodos(String tabla, String lista) in D:\Escritorio\FutureLendingpro\Funciones.cs\Lectura_Base_Datos.cs:line 312
[31/05/2023 09:51:26 p. m.] Error: Error al cargar los promotores: Operación no válida a través de subprocesos: Se tuvo acceso al control '' desde un subproceso distinto a aquel en que lo creó.
[31/05/2023 09:51:26 p. m.] Error: Error al cargar los promotores: Operación no válida a través de subprocesos: Se tuvo acceso al control '' desde un subproceso distinto a aquel en que lo creó.
[31/05/2023 09:54:44 p. m.] Error: Error al cargar los promotores: Operación no válida a través de subprocesos: Se tuvo acceso al control '' desde un subproceso distinto a aquel en que lo creó.
[31/05/2023 09:54:44 p. m.] Error: Error al cargar los promotores: Operación no válida a través de subprocesos: Se tuvo acceso al control '' desde un subproceso distinto a aquel en que lo creó.
[31/05/2023 09:55:34 p. m.] Error: Error al cargar los promotores: Operación no válida a través de subprocesos: Se tuvo acceso al control '' desde un subproceso distinto a aquel en que lo creó.
[31/05/2023 09:55:34 p. m.] Error: Error al cargar los promotores: Operación no válida a través de subprocesos: Se tuvo acceso al control '' desde un subproceso distinto a aquel en que lo creó.
[31/05/2023 10:12:32 p. m.] Error: Error al cargar los promotores: Operación no válida a través de subprocesos: Se tuvo acceso al control '' desde un subproceso distinto a aquel en que lo creó.
[31/05/2023 10:12:32 p. m.] Error: Error al cargar los promotores: Operación no válida a través de subprocesos: Se tuvo acceso al control '' desde un subproceso distinto a aquel en que lo creó.
[31/05/2023 10:12:38 p. m.] Error: System.IndexOutOfRangeException: Could not find specified column in results: Tipo_pago
   at MySql.Data.MySqlClient.ResultSet.GetOrdinal(String name)
   at MySql.Data.MySqlClient.MySqlDataReader.GetString(String column)
   at FutureLending.Lectura_Base_Datos.LectTodos(String tabla, String lista) in D:\Documentos\Trabajos\FutureLendingpro\Funciones.cs\Lectura_Base_Datos.cs:line 291
